basxconnect/core/layouts/editperson.py

```diff
@@ -436,38 +436,31 @@ def revisionstab(request):
 def relationshipstab(request):
     return (
         _("Relationships"),
-        layout.datatable.DataTable.from_model(
-            Relationship,
-            hg.F(
-                lambda c, e: c["object"].relationships_to.all()
-                | c["object"].relationships_from.all()
+        layout.grid.Grid(
```

Comment:
The grid was added in order to have the same alignment of the tab-content as the first tab.

Reply:
Btw, my recent UI-restructuring has good alignment on my 13" laptop but not on a 24" full-HD monitor. I will fix that soon.

Reply:
Ok, I didn't notice anything unaligned on both of my screens. But maybe that is because both (13" and 24") are using the same DPI.

Reply:
Ok, maybe I am mistaken. I will just check again to make sure. It might also be different with Mac, they have some fancy ways of scaling things on different screen sizes...
Anyway, I want to make sure the carbon design is used the same way everywhere. In the official React tutorial they use `Grid(Row(Column(""))))` and if they want to add more rows and columns inside the column, they just start again, which then looks like this: `Grid(Row(Column(Grid(Row(Column("")))))))`